app/controllers/meursing_lookup/steps_controller.rb

```diff
@@ -0,0 +1,41 @@
+module MeursingLookup
+  class StepsController < ApplicationController
+    before_action do
+      @no_shared_search = true
+      @tariff_last_updated = nil
+      clear_session
+    end
+
+    include WizardSteps
+
+    self.wizard_class = MeursingLookup::Wizard
+
+    helper_method :back_link, :last_commodity_code
+
+    private
+
+    def step_path(step = params[:id])
+      meursing_lookup_step_path(step)
+    end
+
+    def wizard_store_key
+      :meursing_lookup
+    end
+
+    def set_page_title
+      @title = "#{@current_step.key} step"
+    end
+
+    def back_link(step_path)
```

Comment:
I don't think the controller should be generating HTML content - thats more the domain of the View layer. I can't see this being used elsewhere in the controller so maybe it could go in a view helper somewhere? Though I'm not sure that saves much over `link_to 'Back', step_path`?

Reply:
I had intended on moving this to a view helper and view spec, actually